frontend/src/component/feedbackNew/FeedbackComponent.tsx

```diff
@@ -131,33 +139,78 @@ const StyledScoreValue = styled('label')(({ theme }) => ({
     },
 }));
 
+const StyledCloseButton = styled(IconButton)(({ theme }) => ({
+    position: 'absolute',
+    right: theme.spacing(2),
+    top: theme.spacing(2),
+    color: theme.palette.background.paper,
+}));
+
 export const FeedbackComponent = () => {
-    const { feedbackData, showFeedback, closeFeedback } = useFeedback();
+    const {
+        feedbackData,
+        showFeedback,
+        closeFeedback,
+    } = useFeedback();
 
     if (!feedbackData) return null;
 
+    const { setToastData } = useToast();
+    const { addFeedback } = useUserFeedbackApi();
+
+
+
+    const onSubmission = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const formData = new FormData(event.currentTarget);
+        const data = Object.fromEntries(formData);
+        await addFeedback(data as any);
```

Comment:
In the example I sent before the author was using zod type to parse data. We can try with the orval generated type. Can it parse data?